### ISSUE TITLE
Add Support For Multiple Locales

### ISSUE THREAD

**Issue by Nibby** [Nibby/Yi#15]
Java/JavaFx has good compatibility with English, but it's good to support a variety of other languages. To start with, this issue is concerned with the implementation of a framework to support text from other languages depending on the system locale settings. 

The primary language will be English. For the MVP, a stretch goal would be to include Chinese translations too. A proper implementation that supports these two languages should serve as a scalable basis for more languages in the future.

This issue is blocked by #30 

### Implementation Rules
- Do not hard code user-facing strings directly in the application itself. Use translation keys instead.
- Maintain translation locale bundles in the project and keep these well documented.
